app-modules/portal/src/Http/Requests/StoreKnowledgeBaseArticleVoteRequest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AidingApp\Portal\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreKnowledgeBaseArticleVoteRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'article_vote' => 'nullable|boolean',
```

Comment:
There are currently request parameters being used in the controller this request is used in that are not mentioned here. Like `articleId`. Please ensure all request parameters are listed here and use `snake_case`.

Also please use array syntax for the rules, NOT string pipe syntax.

Reply:
Done

Reply:
@ketan-canyon I am still seeing these as not being `snake_case`

Reply:
HI @Orrison 
I think you are checking old commit.
its already in snake_case.
<img width="363" alt="{8BF7C839-8A71-4675-94C1-5849FF8EBBFD}" src="https://github.com/user-attachments/assets/a0ab3ea2-1ce1-47c4-b872-31b12cabce0e">

Reply:
@ketan-canyon I am not looking at an old commit. These are still in camelCase.

Reply:
Done now. Sorry because of some my misunderstanding it was not done.